sdk/python/tests/integration/feature_repos/universal/data_sources/bigquery.py

```diff
@@ -74,6 +74,12 @@ def create_data_source(
                 f"{self.gcp_project}.{self.project_name}.{destination_name}"
             )
 
+        # Make all datetime columns timezone aware. This should be the behaviour of
+        # `BigQueryOfflineStore.offline_write_batch`, but since we're bypassing that API here, we should follow the same
+        # rule. The schema of this initial dataframe determines the schema in the newly created BigQuery table.
+        for column in df.columns:
```

Comment:
I might pull this out into a helper function if the maintainers validate this approach.

Reply:
Yeah this looks fine!